docs/proposals/external-ns-service-integration.md

```diff
@@ -1,53 +0,0 @@
-#External Service Integrations
```

Comment:
This can be moved to proposals repo. I can revert change if that's required in this repo.